app/javascript/styles/_index.scss

```diff
@@ -806,6 +806,14 @@ html {
     display: inline-block;
   }
 
+  p.help {
+    margin: 0 0 38px 0;
+  } 
+
+  .Expander_container {
```

Comment:
This seems a bit generic . There's a _component_expander.scss file for system wide/component default changes but the initial question is, should this affect everything in same way. Maybe it's fine because only a width issue. I'm wondering if it's placed near the .flow-detached-group for a reason; perhaps one that indicates it should be inside?

For sake of not holding things up I'm just going to approve everything. If you decide to make some changes I'll try approve those asap.